sechub-developertools/src/main/java/com/daimler/sechub/developertools/admin/ui/action/project/SetProjectMockDataConfigurationAction.java

```diff
@@ -34,7 +34,7 @@ protected void execute(ActionEvent e) throws Exception {
 		
 		DeveloperAdministration administration = getContext().getAdministration();
 		String url = administration.getUrlBuilder().buildSetProjectMockConfiguration(asSecHubId(projectId.get()));
-		administration.getRestHelper().putJSon(url, projectMockConfig.get());
+		administration.getRestHelper().putJSON(url, projectMockConfig.get());
```

Comment:
I think it would be more consistent to call it `putJson()`. I think we call the methods related to JSON `xyzJson`.